style.css

```diff
@@ -1,340 +1,926 @@
+}
+
+.intro__title span {
+    color: #0265DC;
+}
+
+.intro__stats {
+    display: flex;
+    margin-bottom: 30px;
+}
+
+.intro__stat:first-child {
+    margin-right: 40px;
+}
+
+.intro__num {
+    font-size: 44px;
+    line-height: 56px;
+}
+
+.intro__description {
+    max-width: 690px;
+    margin-bottom: 30px;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+
+.intro__title,
+.intro__description,
+.intro__stats,
+.intro > .block__btns {
+    position: relative;
+    z-index: 2;
+}
+
+.btn + .btn {
+    margin-right: 15px;
+}
+
+/* Index page Catalog block */
+
+.catalog {
+    position: relative;
+    z-index: 2;
+}
+
+.catalog__navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 30px;
+}
+
+.catalog__actions,
+.catalog__wrapper {
+    display: flex;
+}
+
+.catalog__actions {
+    order: 3;
+}
+
+.catalog__actions .btn--margin-right {
+    position: relative;
+    margin-right: 32px;
+}
+
+.catalog__actions .btn--margin-right:after {
+    content: '';
+    position: absolute;
+    top: 10px;
+    right: -18px;
+    width: 2px;
+    height: 24px;
+    background-color: #D0D7FB2B;
+}
+
+.catalog__title {
+    position: relative;
+    padding-left: 30px;
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    order: 1;
+}
+
+.catalog__title::before {
+    position: absolute;
+    content: '';
+    top: 3px;
+    left: 0px;
+    width: 24px;
+    height: 24px;
+    background-image: url('images/folder.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.catalog__navigation:after {
+    order: 2;
+    content: '';
+    background: #D0D7FB2B;
+    height: 1px;
+    flex: 1 1 auto;
+}
+
+.catalog__item {
+    padding: 30px;
+    margin-right: 24px;
+    background-color: #1D1C28;
+    transition: all 2s ease;
+}
+
+.catalog__item:hover {
+    background-image: url(images/hover-up-left.png), url(images/hover-down-right.png);            
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: top left, bottom right;
+    transition: all 2s ease;
+}
+
+.catalog__item--hover {
+    background-color: #1D1C28;
+}
+
+.catalog__image {
+    position: relative;
+}
+
+.catalog__price {
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+}
+
+.catalog__name {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    color: #EFF5FA;
+}
+
+/* Index page Blog block */
+
+.blog__wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.blog__post {
+    display: grid;
+    grid-row: span 2;
+    padding: 30px;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    background-color: #1D1C28;
+}
+
+.blog__image {
+    position: relative;
+}
+
+.blog__image img {
+    width: 100%;
+    object-fit: cover;
+}
+
+.blog__post--small {
+    height: 435px;
```

Comment:
Avoid using fixed width and height. Delete height: 435px;

Reply:
done